web/src/signalsHandlers.js

```diff
@@ -172,6 +174,12 @@ export default (store) => (signal) => {
         createSignalAction(goalActions.archiveGoalFully, cellId, payload.data)
       )
       break
+      case SignalType.EditingGoal:
+        // update the state based on the payload of the signal (is_editing)
+        store.dispatch(
+          startTitleEdit(payload.goal_address, payload.editing_agent)
+        )
+        break
```

Comment:
this would get moved up into its own `if -> {}` statement, as mentioned about the line 133 in the other comment